src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+6.1
+---
+* Add the option to extract blank strings instead of a prefixed copy of the translation key with the new `blank()' method in the `TwigExtractor`
```

Comment:
```suggestion
* Add the option to extract blank strings instead of a prefixed copy of the translation key with the new `blank()` method in the `TwigExtractor`
```